spec/components/ny_times/client_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe NyTimes::Builder, type: :client do
```

Comment:
Similar code found in client_spec.rb:3, client_spec.rb:3, client_spec.rb:3 (mass = 330)